cilium-cli/connectivity/check/features.go

```diff
@@ -371,9 +372,7 @@ func (ct *ConnectivityTest) detectFeatures(ctx context.Context) error {
 	for _, ciliumPod := range ct.ciliumPods {
 		// Start with cluster-wide features
 		features := make(features.Set)
-		for k, v := range clusterFeatures {
-			features[k] = v
-		}
+		maps.Copy(features, clusterFeatures)
```

Comment:
Could even further simplify this to:


```suggestion
		features := maps.Clone(clusterFeatures)
```

I understand this change was probably generated by `modernize -fix` which can't do this replacement yet, so more an FYI than a change request.

Reply:
Thanks, updated